kickstarts/partials/post/node.ks.erb

```diff
@@ -0,0 +1,5 @@
+# Get the NodeJS RPM repo setup
+curl --silent --location https://rpm.nodesource.com/setup_6.x | bash -
```

Comment:
looks like the script, actually gets the rpm file itself, and not a .repo.  I'll update the PR